great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -688,6 +689,44 @@ def build_batch_request(
             batching_regex=None,
         )
 
+    @public_api
+    def add_batch_definition_whole_table(self, name: str) -> BatchDefinition:
+        return self.add_batch_definition(
+            name=name,
+            partitioner=None,
```

Comment:
nit, so ignore me: we have defaults on the parent class - I think we're good to trust them.